src/main/java/th/in/gamer/entity/UserEntity.java

```diff
@@ -0,0 +1,43 @@
+package th.in.gamer.entity;
```

Comment:
don't use package named `entity`, it's too generic.
I prefer to use `domain` package